app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -13,23 +13,21 @@ module AuthnJwt
 
       def call
         validate_and_decode_token
-        conjur_id
+        jwt_id
```

Comment:
can we choose only 1 between `jwt_id` and `jwt_identity` ?

Reply:
i will go with jwt_identity